### PR TITLE
optimize production build

### PR DESCRIPTION
**Pull request by TheKevJames**
x400 improvement in load time so far. Imma call that good enough for now.

### CODE REVIEW

File: build/attendee/index.html [hunk 1]
```diff
@@ -19,6 +19,14 @@
 
 <body>
   <div id="app"></div>
+
+  <script src="https://code.jquery.com/jquery-3.1.1.min.js" integrity="sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8=" crossorigin="anonymous"></script>
```
Comment (larajanecka) [inline build/attendee/index.html:23]: ooohhhh good idea

File: package.json [hunk 2]
```diff
@@ -50,10 +44,9 @@
     "webpack-dev-server": "1.16.3"
   },
   "scripts": {
-    "organizer-dev": "NODE_ENV=debug webpack-dev-server --content-base build/organizer --port 8080",
-    "attendee-dev": "NODE_ENV=debug webpack-dev-server --content-base build/attendee --port 8081",
-    "build-prod": "NODE_ENV=prod webpack",
-    "build-stage": "NODE_ENV=staging webpack"
+    "organizer-dev": "webpack-dev-server --content-base build/organizer --port 8080",
+    "attendee-dev": "webpack-dev-server --content-base build/attendee --port 8081",
+    "build": "NODE_ENV=production webpack"
```
Comment (larajanecka) [inline package.json:49]: no stage build?

**Review by larajanecka — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (TheKevJames) [reply package.json:49]: no, every combination of minification + sourcemaps ended up being bigger than just neither. We'll use prod for both dev.calligre and symposium.calligre

File: build/attendee/index.html (hunk 1 above)
Comment (TheKevJames) [reply build/attendee/index.html:23]: I mean, sort of? On the other hand, we now have to manage dependencies in about four places.

**Pull request merged**